### PR TITLE
Add GCC 4.8 build

### PR DESCRIPTION
**Pull request by reidhoch**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ env:
      - CONAN_UPLOAD: "https://api.bintray.com/conan/conan-community/conan"
      - CONAN_STABLE_BRANCH_PATTERN: "release/*"
      - CONAN_UPLOAD_ONLY_WHEN_STABLE: 1 # Will only upload when the branch matches "release/*"
-     - CONAN_DOCKER_32_IMAGES: 1
+     - CONAN_DOCKER_32_IMAGES: 0
```
Comment (lasote) [inline .travis.yml]: Why remove the 32 docker images if you are only adding gcc 4.8 x64?

Comment (reidhoch) [reply .travis.yml]: Good point, I've gone ahead and moved this line out to the specific build for GCC 4.8

File: .travis.yml [hunk 2]
```diff
@@ -22,6 +22,8 @@ osx: &osx
 matrix:
    include:
 
+      - <<: *linux
+        env: CONAN_GCC_VERSIONS=4.8 CONAN_DOCKER_IMAGE=lasote/conangcc48 CONAN_ARCHS=x86_64 CONAN_DOCKER_32_IMAGES=0
```
Comment (lasote) [inline .travis.yml]: Again, maybe I'm not understanding something, but when you use `CONAN_ARCHS=x86_64` you are specifying only 64b builds, so the `CONAN_DOCKER_32_IMAGES=0/1` should be pointless here. Am I missing something?

Comment (reidhoch) [reply .travis.yml]: I had mistakenly thought that the 32bit Docker images were being downloaded, and didn't realize it depended solely on CONAN_ARCHS. I've gone ahead and changed it to remove CONAN_DOCKER_32_IMAGES.

**Pull request merged**